### PR TITLE
Changes from solutions page test review

### PR DESCRIPTION
**Pull request by csmoore**
Changes:

1. Minor edits to the guides based on walk though ( @elinz provided doc versions with Change-Tracking On )
2. Added simple URL checker bat/script to SupportFiles
3. Added exception handler to PortalContentPost.py (needed to get past that step in install)

### CODE REVIEW

**Comment by csmoore** [comment]
The review time has expired on this one, so unless there is objection, I will merge this in a couple days if I don't hear back.

**Comment by conklinbd** [comment]
@elinz can you please take a quick look at this and then merge?

**Comment by elinz** [comment]
@csmoore NOTE: merge for "Added Exception Handler" will not be made. try/exception error print statement reporting incorrect item information. This morning I merged in the changes I made to the PortalContentPost.py using your try/except with correct error info, plus I made an additional modification to the share templates code section.

File: Publish/Portal/PortalContentPost.py [hunk 1]
```diff
@@ -566,17 +566,21 @@ def publish_user_items(portal, username, usercontentpath, old_hostname, new_host
...
-            portal.reassign_item(item['id'], username, foldername)
+				# Reassign item to target owner and folder
+				if os.path.exists(os.path.join(usercontentpath, "folders.json")):
+					os.chdir(usercontentpath)
+					foldersinfo = json.load(open('folders.json'))
+					foldername = publish_get_folder_name_for_item(item, foldersinfo)
+				
+				portal.reassign_item(item['id'], username, foldername)
+			except Exception as err : 
+				print('--> IMPORTANT: Exception on item: ' + str(item))
```
Comment (elinz) [inline Publish/Portal/PortalContentPost.py]: In print statement, the "item" object is not the item that threw the error it's the item previous to error so just change to print item_dir variable which is the item GUID.

Comment (csmoore) [inline Publish/Portal/PortalContentPost.py]: OK I'll pull these files out of this Pull Request

Comment (elinz) [inline Publish/Portal/PortalContentPost.py]: @csmoore I have finished with the review.

Comment (csmoore) [inline Publish/Portal/PortalContentPost.py]: Thanks - note: you can also make a comment on a commit or the main [PR conversation page](https://github.com/Esri/ops-server-config/pull/732) if it doesn't make sense to comment on a line number directly

**Comment by csmoore** [comment]
Guide edits/corrections made from @elinz review - this should be ready for you to merge now

**Pull request merged**